docs/query-api.md

```diff
@@ -0,0 +1,77 @@
+# Query API with Server-sent Events (SSE)
+
+All Timeplus functions is available through RESTful API. To support real-time query result pushed from server to client side, there are two popular solutions, [websocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) and [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events). Timeplus is now leveraging sever-sent events to push real-time query result to the client.
+
+## Event Stream Protocol
+
+An SSE event stream is delivered as a streaming HTTP response: the client initiates a regular HTTP request, the server responds with a custom "text/event-stream" content-type, and then streams the UTF-8 encoded event data.
+
+Here is a sample:
+
+```
+=> Request
+GET /stream HTTP/1.1
+Host: example.com
+Accept: text/event-stream
+
+<= Response
+HTTP/1.1 200 OK
+Connection: keep-alive
+Content-Type: text/event-stream
+Transfer-Encoding: chunked
+
+event: query
+data: {
```

Comment:
There won't be new line or space under `data`section. It is  `{"id":"b3347493-92a2-4bde-bdc5-8ac119199e75","sql":"select * from iot",...}` Maybe we should keep the original format to avoid misunderstading?